monitor/flowgen/bridging/bridging.go

```diff
@@ -111,3 +125,39 @@ func Jobs(networkID netconf.ID, backends ethbackend.Backends, owner common.Addre
 
 	return []types.Job{job1, job2}, nil
 }
+
+// estimateOrderSize checks the current balance of the flowgen EOA and returns
+// the maximal possible order size or nil if the minimal balance is reached.
+func estimateOrderSize(
```

Comment:
nit: rather return `*big.Int, bool, error`, the bool makes result valid status explicit, rather than implicitly encoding it in the value. This is slightly easier to read.